app/views/registrations/_signup.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <div class="row">
+    <div class="col">
+    </div>
+  </div>
+</div>
+<main class="Reg_Main Reg_container">
+  <div class="container Reg_border">
+    <div class="row">
+      <div class="col Reg_AboutYou">
+        <b class="Reg_b_About">About</b>
+        <b class="Reg_b_You">You</b>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <h3>
+          <b>Contact data:</b>
+        </h3>
+      </div>
+    </div>
+    <%= render "registrations/form_registration" %>
```

Comment:
Hier würde ein Partial `form` ausreichen, da du diese Datei bereits unter `registrations` liegen hast.

Reply:
Ich fand das so schöner oder ist das nicht Code konform?

Reply:
Das würde kein anderer Entwickler so erwarten. Standard in Rails ist einfach nur `_form.*`.

Reply:
okay

Reply:
Übrigens, da du bereits hier in `registrations` bist, reicht hier dann einfach <%= render "form" %>.

Reply:
geändert

Reply:
Du musst nicht jedes Mal das hier schreiben, GitHub sagt mir das schon :)